meta/fraction-metadata/src/main/java/org/wildfly/swarm/fractions/FractionUsageAnalyzer.java

```diff
@@ -284,6 +312,13 @@ private void loadDetectorsAndScanners() {
         detectorsLoaded = true;
     }
 
+    private String transformClassSource(String entry) {
+        entry = entry.replace("WEB-INF/classes/", "");
+        entry = entry.replace(".class", "");
+        entry = entry.replace(File.separator, ".");
```

Comment:
Ok, so thanks to this I could fix this case for Windows too, I updated the PR. Seems no File.separator is needed after all. Thanks!